src/components/Item.js

```diff
@@ -0,0 +1,15 @@
+const Item = ({recipeData}) => {
+    console.log(recipeData)
```

Comment:
Helpful for debugging, but remember to remove the console.log before merging into main 👍